### DISCUSSION TITLE
Vue Listbox / Popover with popper.js

### DISCUSSION THREAD

**Discussion by dvh91** [tailwindlabs/headlessui#805] [Help]
Looks like the Vue examples don't have an example with using popper.js or other solution for appending the popover to body to prevent overflow issues.

Is there any known integration with such solution or another?

**Comment by ycs77** [comment]
Hi! I create a package [Headless UI Float](https://github.com/ycs77/headlessui-float), that can be easy to position floating Headless UI elements using Floating UI (is new version Popper.js), support Transition & Portal.

If you need to float Headless UI element can try it out 😊
